### PR TITLE
[CHAD-3691] Formatted fingerprints in a number of Zigbee/ZLL generic DTHs

### PR DESCRIPTION
**Pull request by tpmanley**
Went through a number of the Zigbee and ZLL DTHs and organized the fingerprints by manufacturer. I also corrected deviceJoinName issues in a few of the IKEA fingerprints.

### CODE REVIEW

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 1]
```diff
@@ -22,59 +22,84 @@ metadata {
...
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "0019", manufacturer: "innr", model: "BF 263", deviceJoinName: "Innr Smart Filament Bulb Vintage"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "0019", manufacturer: "innr", model: "BF 265", deviceJoinName: "Innr Smart Filament Bulb"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0019", manufacturer: "innr", model: "AE 260", deviceJoinName: "Innr Smart Bulb"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0019", manufacturer: "innr", model: "BE 220", deviceJoinName: "Innr Smart Flood Light White"
+		fingerprint manufacturer:"innr", model:"RF 265", deviceJoinName: "Innr Smart Filament Bulb White" // raw description: 01 0104 0101 01 09 0000 0003 0004 0005 0006 0008 0B05 1000 FC82 01 0019
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Besides moving things around, the only other change was `mfg`->`manufacturer`

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 2]
```diff
@@ -23,16 +23,29 @@ metadata {
...
+		// INGENIUM
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0B04", outClusters: "0000, 0004", manufacturer: "MEGAMAN", model: "SH-PSUKC44B-E", deviceJoinName: "INGENIUM ZB Smart Power Adaptor", ocfDeviceType: "oic.d.smartplug"
+
+		// Ozom
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", outClusters: "0000", manufacturer: "ClimaxTechnology", model: "PSM_00.00.00.35TC", deviceJoinName: "Ozom Smart Plug", ocfDeviceType: "oic.d.smartplug"
-		fingerprint profileId: "0104", inClusters: "0000, 0702, 0003, 0009, 0B04, 0006, 0004, 0005, 0002", outClusters: "0000, 0019, 000A, 0003, 0406", manufacturer: "Develco Products A/S", model: "Smart16ARelay51AU", deviceJoinName: "Aurora Smart Inline Relay"
+
+		// Philio
 		fingerprint manufacturer: " ", model: "PAN18-v1.0.7", deviceJoinName: "Philio Smart Plug", ocfDeviceType: "oic.d.smartplug" //profileId: "0104", inClusters: "0000, 0003, 0006, 0702", outClusters: "0003, 0019",
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", manufacturer: "SALUS",  model: "SX885ZB", deviceJoinName: "Salus miniSmartplug"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:33]: There were two spaces before `model`

File: devicetypes/smartthings/zll-rgb-bulb.src/zll-rgb-bulb.groovy [hunk 3]
```diff
@@ -25,8 +25,10 @@ metadata {
 		capability "Switch Level"
 		capability "Health Check"
 
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E27 CWS opal 600lm", deviceJoinName: "TRADFRI bulb E27 CWS opal 600lm"
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E26 CWS opal 600lm", deviceJoinName: "TRADFRI bulb E26 CWS opal 600lm"
+
+		// IKEA
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E27 CWS opal 600lm", deviceJoinName: "IKEA TRÅDFRI bulb E27 CWS opal 600lm"
```
Comment (tpmanley) [inline devicetypes/smartthings/zll-rgb-bulb.src/zll-rgb-bulb.groovy:30]: I changed `TRADFRI` -> `IKEA TRÅDFRI` in the `deviceJoinName` for consistency with other DTHs

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 4]
```diff
@@ -22,59 +22,84 @@ metadata {
...
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "0019", manufacturer: "innr", model: "BF 263", deviceJoinName: "Innr Smart Filament Bulb Vintage"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "0019", manufacturer: "innr", model: "BF 265", deviceJoinName: "Innr Smart Filament Bulb"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0019", manufacturer: "innr", model: "AE 260", deviceJoinName: "Innr Smart Bulb"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0019", manufacturer: "innr", model: "BE 220", deviceJoinName: "Innr Smart Flood Light White"
+		fingerprint manufacturer: "innr", model: "RF 265", deviceJoinName: "Innr Smart Filament Bulb White" // raw description: 01 0104 0101 01 09 0000 0003 0004 0005 0006 0008 0B05 1000 FC82 01 0019
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy:51]: Changed `mfr` to `manufacturer` and added spaces after `manufacturer` and `model`

**Comment by tpmanley** [comment]
I called out all the changes that weren't just moving fingerprints around. This is ready for review.

**Review by varzac — DISMISSED** [review]

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy [hunk 5]
```diff
@@ -22,38 +22,49 @@ metadata {
...
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000, FC0F", outClusters: "0019", manufacturer: "OSRAM", model: "CLA60 OFD OSRAM", deviceJoinName: "OSRAM SMART+ LED Classic A60 Dimming"
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "Philips", model: "LWB004", deviceJoinName: "Philips Hue White"
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "Philips", model: "LWB006", deviceJoinName: "Philips Hue White"
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "Philips", model: "LWB007", deviceJoinName: "Philips Hue White"
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "Philips", model: "LWB010", deviceJoinName: "Philips Hue White"
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "Philips", model: "LWB014", deviceJoinName: "Philips Hue White"
+
+		//AduroSmart
```
Comment (dkirker) [inline devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy]: nit: space

File: devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy [hunk 6]
```diff
@@ -29,15 +29,30 @@ metadata {
 
         attribute "colorName", "string"
 
+        // Generic
```
Comment (dkirker) [inline devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy]: Are these all 4 space?

**Review by dkirker — DISMISSED** [review]
Left a couple of comments. Looks good.

**Review by BroderickCarlin — DISMISSED** [review]

File: devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy (hunk 6 above)
Comment (tpmanley) [reply devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy]: They were. I pushed a commit that converts them to tabs

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy (hunk 5 above)
Comment (tpmanley) [reply devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy]: Fixed

File: devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy [hunk 7]
```diff
@@ -34,6 +34,32 @@ metadata {
 		// Generic
 		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0006, 0008, 0300"// deviceId 010C = "White color temperature light"
 
+		// AduroSmart
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy:37]: I realized I missed this DTH in the first round of formatting fixes

**Comment by tpmanley** [comment]
I forced pushed an update which dismissed the approvals. Can someone take another look quick?

**Review by varzac — APPROVED** [review]

**Review by bretschuring — APPROVED** [review]

**Pull request merged**